packages/core/src/components/essentials/tooltip.ts

```diff
@@ -19,6 +19,11 @@ export class Tooltip extends Component {
 
 	tooltip: any;
 	positionService = new Position();
+	isTooltipEnabled = Tools.getProperty(
```

Comment:
this should be somewhere inside in `render()`, where if the user disables the tooltip using an options update, currently the `enabled` value will be unchangeable.

Reply:
Class variable, `isTooltipEnabled` has already been removed.